code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -761,12 +761,12 @@
 /obj/item/melee/transforming/cleaving_saw
 	name = "cleaving saw"
 	desc = "This saw is the tool of choice for the Head Hunter. Capable of switching its reach and attack speed on the fly, it's an incredibly useful weapon for slaying the denizens of the wastes. Animal or human, the saw doesn't judge."
-	force = 30
-	force_on = 60 //force when active
+	force = 10
+	force_on = 30//force when active
```

Comment:
I feel like that is too drastic, you'd probably want to nerf armor penetration and damage to something in-between:
```suggestion
	force = 20
	force_on = 40//force when active
```

Reply:
True, but it's absurdly easy to get

Reply:
Have you learned how to just running over and grabbing it while kiting the mobs?

Reply:
So I'm guessing you haven't seen the dungeon mobs guarding it then
![image](https://user-images.githubusercontent.com/79063506/167916929-03ee8d5f-b737-4ac6-83b0-4b03b00c22e5.png)

Reply:
Yeah, few spiders, kinda easy, get the deathclaw to smash the barriers run in take it, and boom

Reply:
What.... how do you get the deathclaw to smash the barricades, do you see where they are? Unless you're IN that little box, then the deathclaw will have to reason to attack the barricades. So you're forced to fight the dude.